src/Root.cpp

```diff
@@ -720,6 +756,27 @@ void cRoot::BroadcastPlayerListsAddPlayer(const cPlayer & a_Player, const cClien
 }
 
 
+
+
+
+size_t cRoot::GetMaxRAMChunks()
```

Comment:
We need more evidence guided optimization. I say keep in the cpp file unless we have evidance that the lack of inlining is causing a slowdown.